hackyourrepo-app/script.js

```diff
@@ -1,5 +1,62 @@
-"use strict";
+'use strict';
 
-/*
-  Write here your JavaScript for HackYourRepo!
-*/
+//place holder data
+const placeHolderRepos = [
+  {
+    name: 'SampleRepo1',
+    description: 'This repository is meant to be a sample',
+    forks: 5,
+    updated: '2020-05-27 12:00:00',
+  },
+  {
+    name: 'AndAnotherOne',
+    description: 'Another sample repo! Can you believe it?',
+    forks: 9,
+    updated: '2020-06-27 12:00:00',
+  },
+  {
+    name: 'HYF-Is-The-Best',
+    description:
+      "This repository contains all things HackYourFuture. That's because HYF is amazing!!!!",
+    forks: 130,
+    updated: '2020-07-27 12:00:00',
+  },
+];
+
+// Find and store the elements we want to manipulate
+const menuElement = document.querySelector('#selectMenu');
+const repoName = document.querySelector('#repository');
+const repoDescription = document.querySelector('#description');
+const repoForks = document.querySelector('#forks');
+const repoUpdated = document.querySelector('#updated');
+const repoContributors = document.querySelector('#contributors');
+
+//define what you want to happen when the <select> element's value is changed
+function displayRepoInformation(){
+  //select > option > value, starting from cero:
+  const option = menuElement.value; 
+
+  //iterate through the repos 
+  for (let index = 0; index < placeHolderRepos.length; index++) {
+    const element = placeHolderRepos[index];
+
+    //display the repo that matches the selected value (const option)
+    if (option == index) {
```

Comment:
```suggestion
const index = +menuElement.value;
```
Let me save you a for loop.